todo/Controllers/AccountController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using todo.Models;
+using todo.Repositories;
+
+namespace todo.Controllers
```

Comment:
```suggestion
namespace ToDo.Controllers
```
convention cho namespace là dạng CamelCase nhé